surface/src/on_nurbs/triangulation.cpp

```diff
@@ -245,15 +244,14 @@ Triangulation::convertTrimmedSurface2PolygonMesh (const ON_NurbsSurface &nurbs,
     else
     {
       param = pcl::on_nurbs::FittingCurve2dAPDM::findClosestElementMidPoint (curve, vp);
-      param = pcl::on_nurbs::FittingCurve2dAPDM::inverseMapping (curve, vp, param, err, pc, tc, rScale);
+      pcl::on_nurbs::FittingCurve2dAPDM::inverseMapping (curve, vp, param, err, pc, tc, rScale);
```

Comment:
I think we can remove `param = ` in the other branch as well. And make `param` variable local to this block.

Reply:
Good catch!